js-apps/meep-frontend/src/js/containers/exec/dashboard-cfg-detail-pane.js

```diff
@@ -145,6 +149,104 @@ class DashCfgDetailPane extends Component {
     this.onUpdateDashCfg(DASH_CFG_START_TIME, '', null);
   }
 
+  // Callback function for postSeqQuery
+  postSeqQueryCb(error, data, format) {
+    if (error !== null) {
+      this.refreshSeqMetrics = true;
+      return;
+    }
+    var seqMetrics = (data && data.seqMetricString) ? data.seqMetricString : [];
```

Comment:
Should be an empty string instead of an array if string is not provided.

Reply:
Corrected.